### DISCUSSION TITLE
Localization Changes for 3.0

### DISCUSSION THREAD

**Discussion by kmurari103** [iTwin/itwinjs-core#3359] [Q&A] [answered]
Hi there,
I am upgrading my web viewer app to 3.0 and need some guidance on making localization work.
Previously i did follow [this link](https://www.itwinjs.org/learning/frontend/localization/#:~:text=The%20messages%20can%20now%20be%20accessed%20by%20first%20registering%20the%20namespace%2C%20and%20then%20using%20the%20translate%20method%3A) to get it working but having trouble updating to 3.0 way of localization.

```
{
const translationNamespace: I18NNamespace = IModelApp.registerNamespace ("WebViewer");
await translationNamespace.readFinished;

  console.log(IModelApp.i18n.translate("WebViewer:label.pointName")); 
}
```

**Comment by aruniverse** [comment, marked as answer]
Please refer to the [changelog](https://www.itwinjs.org/changehistory/#localization-initialization). 

IModelApp.i18n had been renamed to IModelApp.localization
